### DISCUSSION TITLE
Replica Cluster Across DC

### DISCUSSION THREAD

**Discussion by dbaasvc** [cloudnative-pg/cloudnative-pg#5385] [General]
Dear CloudNativePG,
Your Open Sourced cloudnativepg for running Postgres in Couldnative way is really praise worthy and we are evaluating it for our use cases specially on KOB.  Many KUDOS to the team behind such a nice piece of software .

In an current evaluation , we are trying to get PostgresDB server running across completely 2 independent K8s cluster in 2 different location, specially in the form of Replica Cluster . But we fail to do so as it fails in certificate verification . 
Upon checking documentation specially externalCluster section :

externalClusters:
  - name: cluster-example
    connectionParameters:
      host: cluster-example-rw.default.svc
      user: streaming_replica
      sslmode: verify-full
      dbname: postgres
    # NOTE: if this cluster is created in a different namespace than the main cluster
    # remember to create the `-replication` and `-ca` secrets in the follower namespace
    # before creating the follower cluster
    sslKey:
      name: cluster-example-replication
      key: tls.key
    sslCert:
      name: cluster-example-replication
      key: tls.crt
    sslRootCert:
      name: cluster-example-ca
      key: ca.crt
 
the hostname mentioned here works when the cluster in different namespace but in same k8s cluster but fails to get on to the actual  postgres cluster running in separate k8s cluster and please suggest what should be the host name here . 

The main issue we are stuck at  sslkey,sslcert and sslrootcert all these are self signed for streaming_replica user and in our org across DC communication doesn't allow to have self signed certificate . Could you please guide here how to get this resolved ? 

Your help is highly appreciated and thanks for all your good work done on CloudnativePG

Regards
Debashis

**Comment by radhupr** [comment]
I have similar issue to report. I cannot use selfsigned certs while using a DNS name/LB IP  to replicate a database cluster across DC ( different kubernetes clusters). The option I tried and working is 
`sslmode: verify-ca` - this works but I'm not sure if this will be recommended approach
